js/templates/shared/toggle-button.js

```diff
@@ -0,0 +1,18 @@
+const bel = require('bel');
+
+module.exports = function (isActiveBoolean, klass, dataKey) {
+
+    // make `klass` and `dataKey` optional:
+    klass = klass || '';
+    dataKey = dataKey || '';
+
+    return bel`
+    <button class="toggle-button toggle-button--is-active-${isActiveBoolean} ${klass}"
+            data-key="${dataKey}"
+            type="button">
+        <div class="toggle-button__bg">
+        </div>
+        <div class="toggle-button__knob"></div>
+    </button>`;
```

Comment:
Since this is a `<button>` now, is there has to be a way to avoid the highlighting that happens when it is clicked on. It looks like:

<img width="79" alt="screen shot 2017-05-22 at 4 35 47 pm" src="https://cloud.githubusercontent.com/assets/333669/26327426/6a522886-3f0d-11e7-988c-d8a0c39fdcb9.png">